ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos-device-documentation.j2

```diff
@@ -13,6 +13,8 @@
 {% include 'documentation/name-servers.j2' %}
 {## Domain Lookup #}
 {% include 'documentation/domain-lookup.j2' %}
+{# Clock Timezone #}
```

Comment:
Please change to two ## to make it clear what level of heading is inside the child template.